packages/plugin-core/src/_extension.ts

```diff
@@ -522,4 +524,55 @@ async function showWelcomeOrWhatsNew({
       // no change
       break;
   }
+
+  if (shouldDisplayLapsedUserMsg()) {
+    showLapsedUserMessage();
+  }
+}
+
+function showLapsedUserMessage() {
+  const START_TITLE = "Get Started";
+
+  AnalyticsUtils.track(VSCodeEvents.ShowLapsedUserMessage);
+  MetadataService.instance().setLapsedUserMsgSendTime();
+
+  vscode.window
+    .showInformationMessage(
+      "Get started with Dendron.",
+      { modal: true },
```

Comment:
what happens when a user doesn't have 1.58 of vscode?

Reply:
Actually, it turns out this feature came out long before 1.58.  The only feature added in 1.58 is the ability to add a `detail` prop in the options, which I decided not to utilize.